index.html

```diff
@@ -32,7 +32,105 @@
             <h1 class="carousel-heading">Meet fun people</h1>
             <p class="carousel-paragraph">optional rating of other gamers</p>
           </div>
+          <a class="carousel-control-prev" href="#carouselControls" role="button" data-slide="prev">
+            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+            <span class="sr-only">Previous</span>
+          </a>
+          <a class="carousel-control-next" href="#carouselControls" role="button" data-slide="next">
+            <span class="carousel-control-next-icon" aria-hidden="true"></span>
+            <span class="sr-only">Next</span>
+          </a>
         </div>
+      </header>
+
+      <nav class="navig container-fluid">
+        <ul class="lista row ">
+            <li
+                class="lista_home col-md-3">
+                <a
+                    href="#hero">HOME</a>
+            </li>
+            <li
+                class="lista_about col-md-3">
+                <a
+                    href="#function">ABOUT</a>
+            </li>
+            <li
+                class="lista_authors col-md-3">
+                <a
+                    href="#authors">AUTHORS</a>
+            </li>
+            <li
+                class="lista_extras col-md-3">
+                <a
+                    href="#extras">EXTRAS</a>
+            </li>
+        </ul>
+    </nav>
+    <main>
+    
+    <section class="functions--content">
+        
+        <h1 class="functions--content--header">Functions</h1>
+        
+        <div class="functions--content--container">
+
+          <div>
+
+              <h2 class="functions--content--header">
+                  1. Choose game
+              </h2>
+              <div class="test-img-container--1"></div>
```

Comment:
if it look better in div, let stay with divs :) so remove `test` from class name

Reply:
and I don't know what to do now - Mikolaj promised to provide icons of equal size / scale, which should eliminate the problem and reason for this temporary solution